Userland/Libraries/LibJS/SourceCode.cpp

```diff
@@ -12,23 +12,23 @@
 
 namespace JS {
 
-NonnullRefPtr<SourceCode> SourceCode::create(DeprecatedString filename, DeprecatedString code)
+NonnullRefPtr<SourceCode> SourceCode::create(String filename, String code)
 {
-    return adopt_ref(*new SourceCode(move(filename), move(code)));
+    return adopt_ref(*new SourceCode(std::move(filename), std::move(code)));
```

Comment:
Please leave the `move`s as is, don't add `std` (I know that annoyingly IDEs sometimes autocomplete this)

Reply:
My STL muscle memory must have taken over. Fixed.